### DISCUSSION TITLE
Look up dangerous goods (hazmat) classification by ASIN

### DISCUSSION THREAD

**Discussion by spinneret** [amzn/selling-partner-api-models#3554] [General]
Is it possible to lookup if a product is classified as dangerous goods (hazmat) by ASIN? I reviewed the API reference in detail and couldn't find an endpoint that can be used to retrieve that information.

**Comment by GJ-JG** [comment]
Hi @spinneret , the closest one is to refer [getItemEligibilityPreview](https://developer-docs.amazon.com/sp-api/docs/fbainboundeligibility-api-v1-reference#getitemeligibilitypreview) and check the ```IneligibilityReasonList``` object. 

According to the doc, If the value is between ```FBA_INB_0007```~```FBA_INB_0016``` then it should be related to something Hazmat.

**Reply by spinneret** [reply]
I am interested in getting the hazmat classification for eligible products. The one you referenced applies to ineligible products.

**Reply by XDapps** [reply]
Did you ever have any luck with this?
